changelog/fragments/1758032089-quicker-fleet-check-in.yaml

```diff
@@ -0,0 +1,45 @@
+# REQUIRED
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# REQUIRED for all kinds
+# Change summary; a 80ish characters long description of the change.
+summary: add support for quicker fleet check-in upon component status
```

Comment:
Let's remove this changelog, I don't want anyone outside of us ourselves attempting to use this so best they don't know we support it.

We have had occasional requests for this and we haven't qualified it will work beyond a couple of agents under management.

Reply:
The reason I added it was because of a workflow that was failing, requesting a fragment. Can I overcome that?

Reply:
You can add the `skip-changelog` label on the PR and it will allow you to not have it.